### ISSUE TITLE
[GlobalStep - 4.1 beta 1] Confirmation email to change email address fails to receive. 

### ISSUE THREAD

**Issue by gglobalstep** [woocommerce/woocommerce#26146]
Step to Reproduce:
1. Create a test site using JN. 
2. Complete initial screens and go to dashboard.
3. Click on "Users".
4. Click on "Add User".
5. Enter all the details.
6. From Role drop down select "Customer".
7. Now go to "WooCommerce -> Dashboard-> Customer".
8. Change email address and updated details.
9. Observe that confirmation email to change email address fails to receive.

Actual Result:
Confirmation email to change email address fails to receive.

Expected Result:
Mail should be sent to the updated mail.

**Comment by juliaamosova** [comment]
Hi @gglobalstep,

Thank you for submitting the issue. However, I don't fully understand how to reproduce the issue. 

In particular, after the step below:

>Now go to "WooCommerce -> Dashboard-> Customer".

![customers](https://user-images.githubusercontent.com/19143190/79378032-a150c300-7f2a-11ea-8fd9-f3602edd2d62.jpg)

Should I be going back to Users section and edit the email address there?

![customers2](https://user-images.githubusercontent.com/19143190/79378195-e83eb880-7f2a-11ea-995b-046d3d9b78d5.jpg)

Just to clarify - do you expect for the email confirmation email to be sent to the new customer's email address letting them know that their email address was updated?

**Comment by gglobalstep** [comment]
Hi @juliaamosova 

Yes the user should receive confirmation email upon updating there new email address.
This is what we are expecting.

**Comment by tammullen** [comment]
Hi @gglobalstep 

I checked this in 5.1 changing the email of a customer from the WP edit user screen which I navigated to from the link in the customers screen.

I used WP Mail Logging by MailPoet plugin to confirm the correct email was sent.
![Screenshot_2021-03-25_at_19_33_43](https://user-images.githubusercontent.com/24649833/112532898-4746b780-8da1-11eb-8cfd-17aabdcc591a.png)

Closing this as everything is working as expected.
